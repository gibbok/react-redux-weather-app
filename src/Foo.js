import React from 'react'

const Foo = React.createClass({
  render () {
    return (
      <p onClick={() => {}}>I am not a very smart component...</p>
    )
  }
})

export default Foo
