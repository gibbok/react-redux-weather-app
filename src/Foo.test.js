import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Foo from './Foo'

describe('<Foo />', () => {
  it('render', () => {
    const wrapper = shallow(
      <Foo>
        <p>I am not a very smart component...</p>
      </Foo>
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
