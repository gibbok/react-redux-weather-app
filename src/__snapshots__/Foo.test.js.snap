// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Foo /> renders correctly 1`] = `
<p
  onClick={[Function]}
>
  I am not a very smart component...
</p>
`;
